.eslintrc

```diff
@@ -297,7 +300,7 @@ rules:
     - 6
   max-len:
     - 2
-    - 80
+    - 170
```

Comment:
Please change this back and add edlint rule exceptions where needed instead (see http://eslint.org/docs/user-guide/configuring#configuring-rules)